packages/rollup-plugin-html/src/output/emitAssets.ts

```diff
@@ -57,7 +82,54 @@ export async function emitAssets(
 
       let ref: string;
       let basename = path.basename(asset.filePath);
+      const emittedExternalAssets = new Map();
       if (asset.hashed) {
+        if (basename.endsWith('.css') && options.bundleAssetsFromCss) {
+          let updatedCssSource = false;
+          const { code } = await transform({
+            filename: basename,
+            code: asset.content,
+            minify: false,
+            visitor: {
+              Url: url => {
```

Comment:
does this recursively follow all css files includes (the `@import` ones)?

if yes, looks like `source = Buffer.from(code);` would not update anything but the entry point

Reply:
The code as currently implemented does not _bundle_ css via `@import`. I looked at both esbuild and lightningcss, and they have a `bundle` for css, but it _only_ follows `@import`, and not `url`, which is what we need. So for now I only added support for assets referenced via `url`, afaik you can't add .css files in `url`. In the future we could consider bundling imported css

Reply:
clear, makes sense!